### PR TITLE
[sui-ssr] dymanic rendering per page

### PR DESCRIPTION
**Pull request by carlosvillu**

### CODE REVIEW

File: packages/sui-ssr/server/dynamicRendering/index.js [hunk 1]
```diff
@@ -2,17 +2,16 @@ import path from 'path'
 import fs from 'fs'
 import seoBotDetect from './seoBotDetect'
 
-const {DYNAMIC_RENDERING = false} = process.env
 const INDEX_HTML_PATH = path.join(process.cwd(), 'public', 'index.html')
 
-export default function dinamycRendering(fallback) {
+export default function dinamycRendering(fallback, dynamicsURLS = []) {
```
Comment (midudev) [inline packages/sui-ssr/server/dynamicRendering/index.js]: I know it was already wrong, but it's spelled `dynamic`. Maybe it's a good time to change the function name.

Comment (carlosvillu) [reply packages/sui-ssr/server/dynamicRendering/index.js]: 👍

File: packages/sui-ssr/server/dynamicRendering/index.js [hunk 2]
```diff
@@ -2,17 +2,20 @@ import path from 'path'
...
 import seoBotDetect from './seoBotDetect'
 
-const {DYNAMIC_RENDERING = false} = process.env
 const INDEX_HTML_PATH = path.join(process.cwd(), 'public', 'index.html')
 
-export default function dinamycRendering(fallback) {
+export default function dynamicRendering(fallback, dynamicsURLS = []) {
   return function middleware(req, resp, next) {
-    if (!DYNAMIC_RENDERING) {
+    if (!dynamicsURLS.length || seoBotDetect(req)) {
```
Comment (midudev) [inline packages/sui-ssr/server/dynamicRendering/index.js]: Awesome. I like this middleware code, it's very neat! Congrats! 👏

Comment (carlosvillu) [reply packages/sui-ssr/server/dynamicRendering/index.js]: JAJAJAJA. I HATE YOU!!!

**Pull request merged**